### PR TITLE
CAF-2941: Remove closed-source dependencies from Autoscaler

### PR DESCRIPTION
**Pull request by arogan92**
* Container updated to use java8-with-tini image.
* exec arguments updated to work with tini
* Unnecessary dependencySets removed
* doc-site-compiler updated to use newly open-sourced version
* Updated the <image> tags to push to caf-internal instead of rh7-artifactory

### CODE REVIEW

**Comment by arogan92** [comment]
Developer builds are successful
http://sou-jenkins2.hpeswlab.net:8080/job/Autoscaler/view/Developer/job/Autoscaler~autoscaler~CAF-2941~CI/1/console

**Comment by arogan92** [comment]
Jira:  https://jira.autonomy.com/browse/CAF-2941

File: autoscale-container/pom.xml [hunk 1]
```diff
@@ -177,22 +177,6 @@
                                                 <outputDirectory>config</outputDirectory>
                                             </fileSet>
                                         </fileSets>
-                                        <dependencySets>
-                                            <dependencySet>
```
Comment (dermot-hardy) [inline autoscale-container/pom.xml]: Why is this first dependency set removed?

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
The `tini` dependency is still present --> https://github.com/Autoscaler/autoscaler/blob/e1b2898ea904c5eba26a8c3a2c5bd68780c8c361/autoscale-container/pom.xml#L86

**Comment by arogan92** [comment]
Changes made after feedback

**Review by mulhollandc — APPROVED** [review]

File: autoscale-container/pom.xml [hunk 2]
```diff
@@ -178,19 +172,9 @@
                                             </fileSet>
                                         </fileSets>
                                         <dependencySets>
-                                            <dependencySet>
+                                            <dependencySet>
```
Comment (dermot-hardy) [inline autoscale-container/pom.xml]: Erroneous whitespace added

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

**Pull request merged**